### PR TITLE
Update dependencies and targets Q on sample

### PR DESCRIPTION
**Pull request by rafaeltoledo**

### CODE REVIEW

**Comment by joshschriever** [comment]
@rafaeltoledo thanks for the PR! Can you run `./gradlew test` locally and fix the issues with the Arrow update?

**Comment by rafaeltoledo** [comment]
Sorry, fixed it! :)

File: gordon-plugin/src/main/kotlin/com/banno/gordon/AdbExtensions.kt [hunk 1]
```diff
@@ -43,7 +43,7 @@ internal fun JadbDevice.executeShellWithTimeout(
 internal fun JadbDevice.isTablet(): IO<Boolean> =
     executeShellWithTimeout(20_000, "getprop", "ro.build.characteristics").map { it!!.contains("tablet") }
 
-internal fun JadbDevice.installApk(apk: File, vararg options: String) = fx {
+internal fun JadbDevice.installApk(apk: File, vararg options: String) = bindingConcurrent {
```
Comment (abergfeld) [inline gordon-plugin/src/main/kotlin/com/banno/gordon/AdbExtensions.kt]:
```suggestion
internal fun JadbDevice.installApk(apk: File, vararg options: String) = IO.fx {
```

Using `IO.fx` or just `fx` here better matches the Arrow docs. 

I'm not sure if `bindingConcurrent` is equivalent judging from the source.

File: gordon-plugin/src/main/kotlin/com/banno/gordon/DevicePools.kt [hunk 2]
```diff
@@ -12,7 +12,7 @@ internal data class DevicePool(
     val devices: List<JadbDevice>
 )
 
-internal fun calculatePools(adb: JadbConnection, strategy: PoolingStrategy): IO<List<DevicePool>> = fx {
+internal fun calculatePools(adb: JadbConnection, strategy: PoolingStrategy): IO<List<DevicePool>> = bindingConcurrent {
```
Comment (abergfeld) [inline gordon-plugin/src/main/kotlin/com/banno/gordon/DevicePools.kt]:
```suggestion
internal fun calculatePools(adb: JadbConnection, strategy: PoolingStrategy): IO<List<DevicePool>> = IO.fx {
```

File: gordon-plugin/src/main/kotlin/com/banno/gordon/GordonTestTask.kt [hunk 3]
```diff
@@ -64,7 +64,7 @@ internal abstract class GordonTestTask : DefaultTask() {
 
     @TaskAction
     private fun runTests() {
-        val runTests = fx {
+        val runTests = bindingConcurrent {
```
Comment (abergfeld) [inline gordon-plugin/src/main/kotlin/com/banno/gordon/GordonTestTask.kt]:
```suggestion
        val runTests = IO.fx {
```

File: gordon-plugin/src/main/kotlin/com/banno/gordon/ManifestLoader.kt [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-import arrow.effects.IO
-import arrow.effects.extensions.io.fx.fx
+import arrow.fx.IO
+import arrow.fx.extensions.io.concurrent.bindingConcurrent
 import com.shazam.axmlparser.AXMLParser
 import java.io.File
 import java.util.zip.ZipFile
 
-internal fun getManifestPackage(apk: File): IO<String> = fx {
+internal fun getManifestPackage(apk: File): IO<String> = bindingConcurrent {
```
Comment (abergfeld) [inline gordon-plugin/src/main/kotlin/com/banno/gordon/ManifestLoader.kt]:
```suggestion
internal fun getManifestPackage(apk: File): IO<String> = IO.fx {
```

File: gordon-plugin/src/main/kotlin/com/banno/gordon/ReportFile.kt [hunk 5]
```diff
@@ -13,7 +13,7 @@ internal fun ReportFile.write(directory: File) = IO {
         .absolutePath
 }
 
-internal fun List<ReportFile>.write(directory: File) = fx {
+internal fun List<ReportFile>.write(directory: File) = bindingConcurrent {
```
Comment (abergfeld) [inline gordon-plugin/src/main/kotlin/com/banno/gordon/ReportFile.kt]:
```suggestion
internal fun List<ReportFile>.write(directory: File) = IO.fx {
```

**Comment by rafaeltoledo** [comment]
Yes, you're right @abergfeld! I made some confusing when checking the diffs from Arrow 0.9 to 0.10. Just updated the gist

**Comment by rafaeltoledo** [comment]
Done!

**Pull request merged**